### PR TITLE
Update /appliance homepage text and design and add /appliances/portfolio page

### LINKED ISSUE #7709: [Core appliance pages] Update homepage

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Update /appliance homepage text and design
- Add /appliances/portfolio page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance and http://0.0.0.0:8001/appliance/portfolio
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Copy docs for [overview](https://docs.google.com/document/d/1Fcr5b5Rn06AB89D6DGPmoIgXLxLalijMhwnu_IyUxV4/edit#heading=h.oq1bqfd7kna3) and [portfolio](https://docs.google.com/document/d/1DABafCPzl9WwOmHX8ejkkiUEKXI2fTlF7UXNB0Xhlo0/edit#heading=h.tf6nbxjryj9q) pages

## Issue / Card

Fixes #7709

### CODE REVIEW

File: templates/appliance/index.html [hunk 1]
```diff
@@ -11,109 +11,80 @@
 <section class="p-strip--suru is-dark">
   <div class="row">
     <h1>Ubuntu Appliance portfolio</h1>
```
Comment (steverydz) [inline templates/appliance/index.html:13]: This is prefixed with "The" in the copy doc

File: templates/appliance/index.html [hunk 2]
```diff
@@ -11,109 +11,80 @@
...
-        </div>
-      </a>
-    </div>
-  </div>
-
-  <div class="u-fixed-width">
-    <hr class="p-separator" />
-  </div>
+<section class="p-strip is-bordered">
+  {% with heading="Featured today" %}{% include "/appliance/shared/_featured_appliances.html" %}{% endwith %}
```
Comment (steverydz) [inline templates/appliance/index.html:21]: The heading in the copy doc is just "Featured"

File: templates/appliance/index.html [hunk 3]
```diff
@@ -11,109 +11,80 @@
...
     <div class="col-7">
-      <h2>Try before you Pi</h2>
-      <p class="p-heading--four">Test drive them safely on your laptop</p>
-      <p>Just for fun, you can run a Virtual Ubuntu Appliance on any PC without affecting your existing system. Most of our appliances have a virtual version unless they really do depend on special hardware.</p>
-      <p>No need for a spare disk or board, just follow the steps  and you’ll be up and running quickly. Virtual Ubuntu Appliances work on Ubuntu, Windows and macOS.</p>
+      <h2>
+        Try before you Pi
+      </h2>
+      <p class="p-heading--four">
+        Test drive them safely on your laptop
```
Comment (steverydz) [inline templates/appliance/index.html:77]: Slightly different in the copy doc

File: templates/appliance/index.html [hunk 4]
```diff
@@ -123,21 +94,34 @@ <h2>Try before you Pi</h2>
...
+        App store easy
+      </h2>
+      <p class="p-heading--four">
+        An app or two for extra foo
+      </p>
+      <p>
+        Upgrade your appliance with extra apps. Try a new autopilot. Enjoy the latest game. Each appliance starts with the essential stuff. Add the sauce you like.
+      </p>
+      <p>
+        Enjoy the same huge app selection as Ubuntu, or get a dedicated store with unique apps for your specific thing. Publishers curate the store you see.
```
Comment (steverydz) [inline templates/appliance/index.html:107]: "huge app selection as Ubuntu" is a link to the snap store in the copy doc

File: templates/appliance/index.html [hunk 5]
```diff
@@ -123,21 +94,34 @@ <h2>Try before you Pi</h2>
...
+      </p>
+      <p>
+        Ubuntu Appliances get apps and updates from our global backbone, just like millions of Ubuntu workstations, devices, laptops and servers every day.
+      </p>
+      <p>
+        <a class="p-button p-link--external" href="https://snapcraft.io/store">Visit the Snapstore</a>
```
Comment (steverydz) [inline templates/appliance/index.html:113]: This isn't in the copy doc

File: templates/appliance/index.html [hunk 6]
```diff
@@ -151,60 +135,89 @@ <h2>App store easy</h2>
...
+        Secure by design
+      </h2>
+      <p class="p-heading--four">
+        Crisp. Clean. Carrier-grade. Core.
+      </p>
+      <p>
+        SecureBoot. Full disk encryption. Hardware crypto keys. An ironclad commitment to your peace of mind.
+      </p>
+      <p>
+        We toughened Ubuntu to harden these things. They bring their magic, we bring our mettle. Community or business-backed, both enjoy the same robust, secure Ubuntu Core.
```
Comment (steverydz) [inline templates/appliance/index.html:158]: "same robust, secure Ubuntu Core" is a link to /core in the copy doc

File: templates/appliance/index.html [hunk 7]
```diff
@@ -151,60 +135,89 @@ <h2>App store easy</h2>
...
+      </p>
+      <p>
+        Strict confinement and read-only packages keep bad code locked down. Certs and signatures for tamper-proof provenance. Full containerisation for no loose ends. And we’ve got your back on critical jobs with ten year updates guaranteed.
+      </p>
+      <p>
+        <a class="p-button " href="/core">Learn about Ubuntu Core</a>
```
Comment (steverydz) [inline templates/appliance/index.html:164]: This isn't in the copy doc

**Comment by pmahnke** [comment]
@steverydz thanks, I have either updated the copy doc, or updated the html... the button CTAs came from the design.

File: templates/appliance/index.html (hunk 3 above)
Comment (pmahnke) [reply templates/appliance/index.html]: fixed

File: templates/appliance/index.html (hunk 4 above)
Comment (pmahnke) [reply templates/appliance/index.html:109]: removed from the copy doc

File: templates/appliance/index.html (hunk 5 above)
Comment (pmahnke) [reply templates/appliance/index.html:115]: added to the copy doc

File: templates/appliance/index.html (hunk 6 above)
Comment (pmahnke) [reply templates/appliance/index.html:160]: removed from the copy doc

File: templates/appliance/index.html (hunk 7 above)
Comment (pmahnke) [reply templates/appliance/index.html:166]: added to the copy doc

**Pull request merged**